testsuite/features/secondary/srv_cobbler_buildiso.feature

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2022 SUSE LLC.
+# Licensed under the terms of the MIT license.
+
+@scope_cobbler @scope_iso
+Feature: Cobbler buildiso
+
+  Scenario: Log in as testing user in the cobbler buildiso context
+    Given I am authorized as "testing" with password "testing"
+
+  Scenario: Copy cobbler profiles on the server in the cobbler buildiso context
+    When I copy autoinstall mocked files on server
+
+  Scenario: Create a dummy distro in the cobbler buildiso context
+    Given cobblerd is running
+    Then create distro "testdistro" as user "testing" with password "testing"
+
+  Scenario: Create dummy profiles in the cobbler buildiso context
+    Given cobblerd is running
```

Comment:
I did include it in most steps since Cobbler could possibly fail in between 2 steps but I am not sure how often this could happen. I removed all of them and left just the first one.